packages/eslint-plugin-mdx/src/rules/helpers.ts

```diff
@@ -72,7 +71,7 @@ export const getRemarkProcessor = (searchFrom: string, isMdx: boolean) => {
   }
 
   /* istanbul ignore next */
-  const { plugins = [], settings } = (result.config ||
+  const { plugins = [], settings } = (result?.config ||
```

Comment:
```suggestion
  const { plugins = [], settings } = (result?.config ??
```

Reply:
Hmm, it seems no difference to the result.

Reply:
It many cases `||` and `??` will be the same, `??` makes it clearer that it is intended to catch nullish values specifically, rather than any falsy value.